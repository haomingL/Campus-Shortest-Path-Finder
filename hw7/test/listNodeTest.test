# This test tests the listNode method
LoadGraph people people.tsv

ListNodes people