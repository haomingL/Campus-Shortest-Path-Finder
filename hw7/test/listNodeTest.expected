# This test tests the listNode method
loaded graph people

people contains: Chenfan-Sun Haoming-Liu Juan-Cai Wenjun-Zhou Xinyi-Xu Yingkai-Wang Youying-Lin